src/content/docs/tutorial-service-level-mgmt/establish-output-sli.mdx

```diff
@@ -0,0 +1,100 @@
+</Callout>
+
+    </Step>
+    <Step>
+## Create your baseline
+
+You must also establish a baseline to implementat service levels. Creating a baseline allows you to measure the current performance of a service, then use service level reports to find out if you're hitting that baseline or degrading. You can create a baseline for almost any dataset; however, you use different formulas and recommendations in different use cases.
+
+When you create service levels, you should start with the output performance of your applications. You can use response times (latency) and percentage of non-errors (success). When you do this, you create a reliability health metric. Seasonality and peak usage doesn't handicap your performance, and the more history you include in your measurement, the more likely you're including different codebases from releases. Previous deployments, no matter how small, could skew your results.
+
+We recommend a history of one to two weeks of performance data to establish a good baseline. Here's an example NRQL query for the recommended target of a seven day service level objective for latency:
+
+```
+FROM Transaction SELECT percentile(duration, 95) AS 'Latency Baseline SLI' WHERE appName='Order-Processing' SINCE 1 WEEK AGO
+```
+
+<img
+  alt="Latency Baseline"
+  src={apmLatencyBaseline}
+/>
+
+For a successful (error-free) baseline, try the following query. Remember to substitute `Order-Processing` for your own application name.
+
+```
+FROM Transaction SELECT percentage(count(*), WHERE error is false) AS 'Success Baseline SLI' SINCE 1 WEEK AGO WHERE appName='Order-Processing'
+```        
+    </Step>
+    <Step>
+## Create your service level
+
+**Note:** If you don't see the **Add a service level** button, check with your New Relic administrator about your permissions.
+
+Our platform automatically calculates the recommended <InlinePopover type="apm" /> and browser baselines for you. The identify your service section above shows you how to find your application APM data. You'll see #2 in the screenshot in that same section, called "Service levels." Find your application APM data and click **Service levels**, and you should see the view below:
+
+<img
+  alt="Service levels start from APM"
+  src={apmSLStartFromApm}
+/>
+
+Click **Add baseline service level objectives** and you'll have both your latency SLI and success SLI, as well as their respective objectives, created for you. You can view and change all the settings by clicking the ellipses icon in the upper-right corner of each SLO scorecard.
```

Comment:
this section seems to be missing a screenshot for the last step after you click "Add a service level." I'd just add something short that describes the flow for this UI just to round out the tutorial.

Reply:
I'm not sure I see where it's missing but I will check on this and see what I can add to further describe the screen. Thank you!